accelerator/config.c

```diff
@@ -70,6 +72,13 @@ status_t cli_core_set(char* conf_file, ta_config_t* const ta_conf, iota_config_t
       cache->port = atoi(value);
       break;
 
+    // DB configuration
+    case DB_ENABLE_CLI:
+      db_service->service_enable = (toupper(value[0]) == 'T');
```

Comment:
Didn't you think it is confusing to express the statement `db_service->service_enable`?  I would prefer `db_service->status`.

Reply:
I did hesitate to use `db_service->service_enable`. Actually, `service_enable` was first putting in `ta_core_t` to be the status of TA db-connection. I move it into `db_service_t` because of the increment of function parameters. 
If naming it as `db_service->status`, should I add an `enum` to define the status or simply add comments around the definition of `db_service->status` ?
I am afraid of the value true/false of `db_service->status` may be confusing.

Reply:
BTW, `db_service->enabled` sounds pretty fine as well.